### ISSUE TITLE
Project submit form not working

### ISSUE THREAD

**Issue by amacgregor** [Vinai/magehack-voting#10]
When trying to submit the form nothing happens and not visible errors.

**Comment by Vinai** [comment]
Oh, yeah...

```
    valid = $scope.newProject.title.length > 3;
    valid = valid && $scope.newProject.description.length > 20;
```

Need to display some indicator :) thx for pointing that out

**Comment by Vinai** [comment]
Fixed in 64c0ae150d6de5e637f0342cc092e5b5e629c188
